R/check-mac.R

```diff
@@ -0,0 +1,71 @@
+#' Check macOS package
+#'
+#' This function works by bundling source package, and then uploading to
+#' <https://mac.r-project.org/macbuilder/submit.html>.  Once building is
+#' complete you'll receive a link to the built package in the email address
+#' listed in the maintainer field.
+#'
+#' @template devtools
+#' @inheritParams pkgbuild::build
+#' @param email An alternative email to use, default `NULL` uses the package
+#'   Maintainer's email.
+#' @param quiet If `TRUE`, suppresses output.
+#' @param ... Additional arguments passed to [pkgbuild::build()].
+#' @family build functions
+#' @return The url with the check results (invisibly)
+#' @export
+check_mac_release <- function(pkg = ".", args = NULL, manual = TRUE, email = NULL, quiet = FALSE, ...) {
+  check_dots_used(action = getOption("devtools.ellipsis_action", rlang::warn))
+
+  pkg <- as.package(pkg)
+
+  if (!is.null(email)) {
```

Comment:
The mac builder doesn't seem to use the email or send you results via email, so it is possible we should just remove this functionality for this. What do you think?

Reply:
Yeah, just remove I think.